sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/helpers/SamplesArguments.java

```diff
@@ -24,19 +26,22 @@ public SamplesArguments(String[] args) {
         Option clientId = new Option("c", CLIENT_ID, true, "AAD Client Id");
         Option clientSecret = new Option("s", CLIENT_SECRET, true, "AAD Client Secret");
         Option logLevel = new Option("l", LOG_DETAIL_LEVEL, true, "Http logging detail level \n 0 -> NONE \n 1 -> BASIC \n 2 -> HEADERS \n 3 -> BODY \n 4 -> BODY_AND_HEADERS");
+        Option eventRouteEndpointName = new Option("e", EVENT_ROUTE_ENDPOINT_NAME, true, "(Event route sample only) The name of an existing event route endpoint");
```

Comment:
I had to add this extra command line parameter because, in order to create an event route, you must have one pre-existing event route endpoint. Creating a new event route endpoint for the user would require the control plane SDK.

Reply:
It is marked as optional both because other samples don't need this value, but also because the event route sample can still do some operations without this value